clayui.com/src/styles/_404-page.scss

```diff
@@ -0,0 +1,66 @@
+.clay-404-page-container {
+	background-image: url('../../static/images/home/bg_white.svg');
+
+	header {
+		background-color: #fff;
+
+		img {
+			width: 32px;
+		}
+	}
+
+	.navbar {
+		width: 100%;
+	}
+
+	.list-inline {
+		a {
+			background-color: $brand-primary;
+			color: #fff;
+			transition: all 0.3s ease-in-out;
+
+			.githubImage {
+				width: 16px;
+				height: 16px;
+				background-size: cover;
+				background-image: url('../../static/images/home/GitHub-Mark-64px.svg');
+			}
+
+			&:hover {
+				background-color: $brand-dark;
+				color: $brand-light;
+				box-shadow: 0 0 0 !important;
+
+				svg {
+					color: $brand-light;
+				}
+			}
+		}
+	}
+
+	.main-content {
+		transform: translateY(-66px);
+
+		@media (max-width: $screen-md) {
+			flex-direction: column;
+			margin: auto 48px !important;
+		}
+
+		@media (max-width: $screen-sm) {
+			margin: auto 24px !important;
+			transform: translateY(0);
+		}
+
+		h1 {
```

Comment:
instead of targeting `h1` maybe it's better to target `.display-3`? From https://www.w3.org/WAI/tutorials/page-structure/headings/, Headings communicate the organization of the content on the page. We should try not to scope styles to `h` tags because we can have multiple `h1` tags on a page that have varying font sizes, colors, etc.

Reply:
> we can have multiple h1 tags on a page that have varying font sizes, colors, etc.

Multiple h1 tags is valid HTML, but traditionally a discouraged practice 😂

Reply:
Sure, even tho there's only one h1 in there and same thing _(we can have multiple tags...)_ could be said about `.display-3`, but yea, no problem switching the selector around.

Reply:
This was a in general comment. We have styles that rely on `h1-h6` selectors in portal. Any change to `h2` for example would break the layout.